schema/me/append_conversation_thread.js

```diff
@@ -35,25 +47,31 @@ export default mutationWithClientMutationId({
     },
   },
   outputFields: {
-    message: {
-      type: MessagePayloadType,
-      resolve: message => message,
+    payload: {
+      type: AppendConversationThreadMutationPayload,
+      resolve: data => data,
     },
   },
   mutateAndGetPayload: ({ id, from, to, body_text }, request, { rootValue: { accessToken } }) => {
     if (!accessToken) return null
-    return gravity
-      .with(accessToken, { method: "POST" })("me/token", {
-        client_application_id: IMPULSE_APPLICATION_ID,
-      })
+    let impulseToken
+    return getImpulseToken(accessToken)
       .then(data => {
+        impulseToken = data.token
         return Promise.resolve(
-          impulse.with(data.token, { method: "POST" })(`conversations/${id}/messages`, {
+          impulse.with(impulseToken, { method: "POST" })(`conversations/${id}/messages`, {
             to: ["partner@example.com"],
```

Comment:
can we still read the `to` parameter from the query and hardcode a value in emission instead?
That way we can test staging emails without having to redeploy metaphysics